package.json

```diff
@@ -51,6 +52,7 @@
     "jest-cli": "^24.8.0",
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1",
+    "react": "^16.9.0",
```

Comment:
nit: react should probably be a dependency instead of devDependency. This matters for people who use @openmrs/esm-api as an npm dependency (which will be a thing for packaging distros)